core/storage/exploration/gae_models.py

```diff
@@ -232,6 +232,10 @@ class ExplorationModel(base_models.VersionedModel):
     # exploration.
     correctness_feedback_enabled = datastore_services.BooleanProperty(
         default=False, indexed=True)
+    # An boolean indicating whether further edits can be made to the
+    # exploration.
+    edits_allowed = datastore_services.BooleanProperty(
```

Comment:
I'm not really sure about the naming I've used here. This setting applies for exps that are published.

Reply:
Thanks for flagging! I think the naming you've chosen is great. 

However, I don't understand why "this setting applies for exps that are published". Firstly, if that's the case, that should be explained in the comment in the code (or, better, in the name). Secondly, though -- I think this setting should apply regardless of publication status.